pom.xml

```diff
@@ -9,7 +9,7 @@
   <artifactId>hockeyapp</artifactId>
   <name>HockeyApp Plugin</name>
   <description>Uploads iOS, Mac or Android apps to hockeyapp.net</description>
-  <version>1.2.1-SNAPSHOT</version>
+  <version>1.2.2</version>
```

Comment:
Current official release is 1.2.0 so 1.2.1-SNAPSHOT is already different ... going to a release version is only done when actually releasing, so for purposes of PR this should be reverted.